pipeline/model/model.py

```diff
@@ -0,0 +1,64 @@
+"""
+Train an item-to-item collaborative filtering model based on similarity matrices
+"""
+
+from itertools import chain
+from pyspark.mllib.linalg.distributed import RowMatrix
+from pyspark.mllib.linalg import Vectors
+from pyspark.ml.feature import CountVectorizer
+from pyspark.sql.functions import col, collect_list, create_map, lit
+from pyspark.sql import SparkSession
+from pyspark import SparkContext
+
+def get_similarity_dataframe(cursor):
+    """
+    Compute a similarity matrix from the dependency table
+    :param cursor: Database cursor
+    :return: Spark DataFrame
+    """
+
+    SC = SparkContext("local[1]", "pkgpkr")
+
+    # Load the raw data into Spark
+    cursor.execute("SELECT * FROM dependencies")
+    DEPENDENCIES = cursor.fetchall()
```

Comment:
why all of those are capitalized (thru line63? none are constants, is it something pyspark specific style?

Reply:
Fixed